src/pu-utils.c

```diff
@@ -376,7 +380,7 @@ pu_bootpart_enable(const gchar *device,
     g_return_val_if_fail(bootpart <= 2, FALSE);
     g_return_val_if_fail(error == NULL || *error == NULL, FALSE);
 
-    cmd = g_strdup_printf("mmc bootpart enable %u 0 %s", bootpart, device);
+    cmd = g_strdup_printf("mmc bootpart enable %u %u %s", bootpart, boot_ack, device);
```

Comment:
`gboolean` is of type `int`, not `unsigned int`:
```suggestion
    cmd = g_strdup_printf("mmc bootpart enable %u %d %s", bootpart, boot_ack, device);
```